arch/arm64/boot/dts/qcom/sm7150-xiaomi-sweet_k6a.dts

```diff
@@ -105,7 +105,7 @@
 		reg = <0x5d>;
 		interrupt-parent = <&tlmm>;
 		interrupts = <9 IRQ_TYPE_NONE>;
-		pinctrl-0 = <&ts_int_active &ts_reset_active>;
```

Comment:
This reset is required. Give me at least one reason why this should be dropped

Reply:
U remember the irq head checksum error in dmesg, this reset cause it
And downstream dtb doesn't have reset

Reply:
> U remember the irq head checksum error in dmesg, this reset cause it And downstream dtb doesn't have reset

Sorry, I don't remember. Downstream uses this anyway. Downstream has in one pinctrl node both pins at once. This is fundamentally wrong, so it is corrected here. In fact, the irg pin should have an input only configuration, and the reset pin should have an output configuration

Reply:
![Screenshot from 2024-04-29 21-02-08](https://github.com/sm7150-mainline/linux/assets/47786161/ef4e7db3-2dff-4ad5-9162-a0a34a55e755)
As you can see gpio8 and gpio9. In mainline these are irq and reset which have been splitted

Reply:
I actually don't believe such a high current is needed, perhaps we could test all devices at 8mA instead of 16mA. I'm almost 99% sure that this will work, tests are just always needed. @Gelbpunkt

Reply:
Xiaomi didnt want to bother. But this causes high load on gpio

Reply:
> I actually don't believe such a high current is needed, perhaps we could test all devices at 8mA instead of 16mA. I'm almost 99% sure that this will work, tests are just always needed. @Gelbpunkt

Yeah 16 is a lot. I'll try 8 when I have some time, it will probably work fine